### DISCUSSION TITLE
Forcefully run http only even ssl certs are provided

### DISCUSSION THREAD

**Discussion by rajeshisnepali** [nginx-proxy/nginx-proxy#1704] [Q&A] [answered]
For local development, I generate local certificates via [mkcert](https://github.com/FiloSottile/mkcert) and mount my certs at `/etc/nginx/certs` in the container. 

`https://site.local` works fine but want to run `http://site.local`. It always redirects me to **https**.

**Comment by buchdag** [comment, marked as answer]
Hi, you'll have to use the `HTTPS_METHOD` environment variable:

https://github.com/nginx-proxy/nginx-proxy#how-ssl-support-works

> To serve traffic in both SSL and non-SSL modes without redirecting to SSL, you can include the environment variable `HTTPS_METHOD=noredirect` (the default is `HTTPS_METHOD=redirect`).  You can also disable the non-SSL site entirely with `HTTPS_METHOD=nohttp`, or disable the HTTPS site with `HTTPS_METHOD=nohttps`. `HTTPS_METHOD` can be specified on each container for which you want to override the default behavior or on the proxy container to set it globally. If `HTTPS_METHOD=noredirect` is used, Strict Transport Security (HSTS) is disabled to prevent HTTPS users from being redirected by the client.  If you cannot get to the HTTP site after changing this setting, your browser has probably cached the HSTS policy and is automatically redirecting you back to HTTPS.  You will need to clear your browser's HSTS cache or use an incognito window / different browser.
